Gds/src/fprime_gds/executables/fprime_cli.py

```diff
@@ -34,7 +34,8 @@ def add_connection_arguments(parser: argparse.ArgumentParser):
         "--dictionary",
         type=str,
         default=None,
-        help='path from the current working directory to the "<project name>Dictionary.xml" file for the project you\'re using the API with; if unused, tries to search the current working directory for such a file',
+        help='path from the current working directory to the "<project name>Dictionary.xml" file for the project'
+             'you\'re using the API with; if unused, tries to search the current working directory for such a file',
```

Comment:
Same as above. I prefer to not separate the strings into two quotes on separate line. Might be better to revert it back to what they were.